tests/test_config.py

```diff
@@ -18,14 +18,26 @@
 import unittest
 from argparse import Namespace
 from copy import deepcopy
+from math import log2
 from typing import Any, Dict, List, Optional
-from unittest.mock import patch
+from unittest.mock import MagicMock, patch
 
+from model_analyzer.analyzer import Analyzer
 from model_analyzer.cli.cli import CLI
+from model_analyzer.config.generate.config_parameters import (
+    ParameterCategory,
+    ParameterType,
+)
 from model_analyzer.config.input.config_command_profile import ConfigCommandProfile
 from model_analyzer.config.input.config_command_report import ConfigCommandReport
 from model_analyzer.config.input.config_defaults import (
```

Comment:
Would it maybe make sense to import than config_defaults, then use all of the below constants? That would make this section more concise. It seems like you'd be using all/most of them anyway.

Reply:
Oh, I mean `import model_analyzer.config.input.config_defaults as config_defaults` or equivalent. Not `from model_analyzer.config.input.config_defaults import *`.

Reply:
Oh, yes you could do that but then the names get longer when they're used. My preference is to call them out explicitly.